Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -25,11 +27,14 @@ public class JiraCommitHintPlugin : GitPluginBase, IGitPluginForRepository
         private static readonly TranslationString EmptyQueryResultCaption = new("First Task Preview");
 
         private const string DefaultFormat = "{Key} {Summary}";
+        private const string BASIC = "Basic";
+        private const string JWT = "JWT";
         private Jira? _jira;
         private string? _query;
         private string _stringTemplate = DefaultFormat;
         private readonly BoolSetting _enabledSettings = new("Jira hint plugin enabled", false);
         private readonly StringSetting _urlSettings = new("Jira URL", @"https://jira.atlassian.com");
+        private readonly ChoiceSetting _authTypeSettings = new("Auth type", new List<string> { BASIC, JWT });
```

Comment:
Hide username if PAT is used, should change label for pat/pw too.
Suggest to add PAT as the default, unless username is added already

Reply:
Yeah, I'd like to make some optimize, before next tuesday.